main.py

```diff
@@ -15,102 +15,100 @@
 SUPPORT_SERVER = "https://discord.gg/CgH6Sj6"
 INVITE = "https://discord.com/api/oauth2/authorize?client_id=715545167649570977&permissions=980675863&scope=bot"
 
-client = commands.Bot(commands.when_mentioned_or(PREFIX), case_insensitivity=True, owner_id=688275913535914014)
-
-status = [
-    "😁Working At The Codin' Hole! Join me at https://discord.gg/aYF76yY",
-    "▶Check out My Creator's Youtube channel : https://www.youtube.com/channel/UC3S4lcSvaSIiT3uSRSi7uCQ/",
-    f"Ping me using {PREFIX}help",
-    "Official Instagram of My Creator ❌ https://instagram.com/the.codin.hole/",
-    "Ready To Work and Get Worked! My Github 🔆 https://github.com/janaSunrise",
+extensions = [
+    "cogs.codesandbox",
+    "cogs.commands",
+    "cogs.converters",
+    "cogs.custom",
+    "cogs.emotes",
+    "cogs.events",
+    "cogs.fun",
+    "cogs.games",
+    "cogs.infog",
+    "cogs.moderation",
+    "cogs.study",
+    "cogs.sudo",
+    "cogs.support",
+    "cogs.tools",
 ]
+# "cogs.coding"
+
+
+class Bot(commands.Bot):
+    def __init__(self, *args, **kwargs) -> Bot:
+        super().__init__(*args, **kwargs)
+        self.status = cycle(
+            [
+                "😁Working At The Codin' Hole! Join me at https://discord.gg/aYF76yY",
+                "▶Check out My Creator's Youtube channel : https://www.youtube.com/channel/UC3S4lcSvaSIiT3uSRSi7uCQ/",
+                f"Ping me using {PREFIX}help",
+                "Official Instagram of My Creator ❌ https://instagram.com/the.codin.hole/",
+                "Ready To Work and Get Worked! My Github 🔆 https://github.com/janaSunrise",
+            ]
+        )
+        self.first_on_ready = True
+
+    async def on_ready(self) -> None:
+        if self.first_on_ready:
+            self.change_status.start()
+            self.fist_on_ready = False
+            self.hw = self.get_user(715545167649570977)
+            self.log_channel = self.get_channel(704197974577643550)
```

Comment:
log channel id should also be defined in [**`constants.py`**](https://github.com/The-Codin-Hole/HotWired-Bot/blob/6e776c3b99088e27767ae2bafaa1f41d9bc93a23/cogs/utils/constants.py)

Reply:
Yeah. However, I only worked on `main.py`, and it was defined like this, so I kept it

Reply:
That's fine it was defined this way because it wasn't in constants.py before that, but now that it is, we can use it directly it shouldn't be hard to do that.

Reply:
I Did actually